src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  let stop = 1;
+  const pages = getNumbers(1, stop).map(n => `${n}`);
+
+  function getPreparePages(pageChange: number) {
+    let preparePages = [...pages];
+
+    if (pageChange) {
+      stop = Math.ceil(total / perPage);
+      preparePages = getNumbers(1, stop).map(n => `${n}`);
+    }
+
+    return preparePages;
```

Comment:
```suggestion
     const stop = Math.ceil(total / perPage);
     const preparePages = getNumbers(1, stop);

    return preparePages;
```
looks like we can simplify it